adserver/views.py

```diff
@@ -782,6 +782,12 @@ def handle_action(self, request, advertisement, offer, publisher):
                 request, self.impression_type, publisher=publisher, offer=offer
             )
 
+            # Update the publisher daily earn for cap calculations
+            if self.impression_type == CLICKS and advertisement.flight.cpc:
+                publisher.increment_daily_earn(float(advertisement.flight.cpc))
+            if self.impression_type == VIEWS and advertisement.flight.cpm:
+                publisher.increment_daily_earn(float(advertisement.flight.cpm) / 1000)
```

Comment:
I do wonder if this will slowly drift given the float nature of it? Not a huge deal, but interesting.

Reply:
Floats should be accurate enough even to 4-5 decimals. I thought about it but I didn't think it would be an issue.

Reply:
Yea, it's not billing data, so not important to get perfect.